src/common/transformations/src/transformations/op_conversions/convert_sequences_to_tensor_iterator.cpp

```diff
@@ -42,7 +42,13 @@ ngraph::Output<ngraph::Node> get_masked_value(const std::shared_ptr<ngraph::opse
     body_params.push_back(aggregated_Y_h_body_param);
 
     // Create mask node deciding whether or not to mask batch data.
-    auto batch_seq_length = ngraph::builder::opset1::legacy_broadcast_for_binary_operation(data, seq_lengths, 0);
+    auto data_shape = ngraph::op::util::make_try_fold<ngraph::opset5::ShapeOf>(data);
+    auto axis = ngraph::opset5::Constant::create(data_shape->get_element_type(), {1}, {0});
```

Comment:
Please use at least new ov namespace

Reply:
this is for consistency with the whole transformation